model/model.js

```diff
@@ -53,4 +54,19 @@ function doesArticleExist(article_id){
     })
 }
 
-module.exports = { fetchTopics, fetchArticleById, fetchArticles, fetchCommentsByArticleId, doesArticleExist }
\ No newline at end of file
+function insertCommentsByArticleId(article_id, username, body){
+    const votes = 0
+    const created_at = new Date()
+    const myNestedArray = [[body,username, article_id, votes, created_at ]]
+    const sqlStr = format(`INSERT INTO comments
+    (body, author, article_id, votes, created_at)
```

Comment:
You shouldn't need to add the `votes` or `created at` into your SQL query. 
Make sure you think about SQL injection here too - is this currently preventing it?